content/docs/operating/storage.md

```diff
@@ -86,3 +86,12 @@ storage directory:
    1. Stop Prometheus.
    1. `rm -r <storage path>/*`
    1. Start Prometheus.
+
+## Hash collisions
+
+Prometheus uses 64-bit fingerprints to identify time series. On a
+large server with serveral million time series, the chance of a hash
+collision is about one to one million (assuming the FNV-1a hash
+function works well). We expect that the probability of a data
```

Comment:
"that the probability ... to be higher than that" -> either lose the "that" or the "to"